tests/http_basicauth.test.js

```diff
@@ -0,0 +1,29 @@
+import child_process from "child_process";
+import fs from "fs";
+import http from 'http';
+
+const server = http.createServer((req, res) => {
+  res.writeHead(200);
+  res.write("Hi!");
+  res.end();
+});
+
+beforeAll(() => {
+  server.listen(9998, '0.0.0.0');
+});
+
+test("test that http basic auth works", async () => {
+
+  child_process.execSync(
+    "docker run -v $PWD/test-crawls:/crawls webrecorder/browsertrix-crawler crawl --url http://host.docker.internal:9998/ --httpBasicAuth foo:bar --collection basicauth-test --statsFilename stats.json",
```

Comment:
Try without the trailing slash after the port?

```suggestion
    "docker run -v $PWD/test-crawls:/crawls webrecorder/browsertrix-crawler crawl --url http://host.docker.internal:9998 --httpBasicAuth foo:bar --collection basicauth-test --statsFilename stats.json",
```